AUTHORS

```diff
@@ -26,7 +26,6 @@ Eugene Zhuk <eugene.zhuk@gmail.com>
 Evgeny Safronov <division494@gmail.com>
 Federico Ficarelli <federico.ficarelli@gmail.com>
 Felix Homann <linuxaudio@showlabor.de>
-Gergő Szitár <szitar.gergo@gmail.com>
```

Comment:
you can leave this in, i think. we should still credit the original contribution.

Reply:
Oh, this isn't intentional, i only used github builtin "revert" feature.
Let be fix this up..